code/style.css

```diff
@@ -1,7 +1,825 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
 
-.active + .answer {
-  /* some styling here */
+  /* backgorund animation */
+  background: linear-gradient(270deg, #aef0df, #d9b7d6, #fbd880);
+  background-size: 400% 400%;
+
+  -webkit-animation: AnimationName 30s ease infinite;
+  -moz-animation: AnimationName 30s ease infinite;
+  animation: AnimationName 30s ease infinite;
+}
+
+@-webkit-keyframes AnimationName {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+@-moz-keyframes AnimationName {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+@keyframes AnimationName {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+p {
+  color: #fff;
+}
+
+button {
+  border: none;
+  cursor: pointer;
+}
+
+button:focus {
+  outline: 2px solid #fff;
+  outline-offset: 4px;
+}
+
+a:focus {
+  outline: 2px solid #fff;
+}
+
+/************ HEADER ************/
+
+/* video as header */
+
+header {
+  position: relative;
+  overflow: hidden;
+  background-size: cover;
+  height: 100vh;
+}
+
+video {
+  height: 100vh;
+  width: 100%;
+  object-fit: cover;
+  position: absolute;
+
+  /* center */
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+picture {
+  display: none;
+}
+
+/* navbar */
+
+ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  color: #fff;
+  padding: 0;
+}
+
+li {
+  margin: 5px 10px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  /* align-items: center; */
+  position: relative;
+}
+
+@keyframes shake {
+  20% {
+    transform: translateY(5px);
+  }
+}
+
+.nav-bar {
+  text-transform: uppercase;
+  font-size: 16px;
+}
```

Comment:
A good practise is to have the keyframes after you've named the animation, so here the `-nav-bar:hover` should be  above the keyframes but still great work 🌻